experimental/dds/tree2/src/feature-libraries/default-field-kinds/defaultFieldKinds.ts

```diff
@@ -38,6 +38,7 @@ export const noChangeHandler: FieldChangeHandler<0> = {
 	codecsFactory: () => noChangeCodecFamily,
 	editor: { buildChildChange: (index, change) => fail("Child changes not supported") },
 	intoDelta: (change, deltaFromChild: ToDelta): Delta.FieldChanges => ({}),
+	relevantRemovedTrees: (change): Iterable<Delta.DetachedNodeId> => [],
```

Comment:
Agreed for the `get` prefix.

I think "referenced" is problematic because a tree could be referenced but not relevant (in which case we don't want to include it in the refreshers). For example, the sequence implementation currently makes the conservative choice of including some referenced trees that may not be relevant (see the comment there).